doc/source/user/depending_on_numpy.rst

```diff
@@ -0,0 +1,149 @@
+.. _for-downstream-package-authors:
+
+For downstream package authors
+==============================
+
+This document aims to explain some best practices for authoring a package that
+depends on NumPy.
+
+
+Understanding NumPy's versioning and API/ABI stability
+------------------------------------------------------
+
+NumPy uses a standard, :pep:`440` compliant, versioning scheme:
+``major.minor.bugfix``. A *major* release is highly unusual (NumPy is still at
+version ``1.xx``) and if it happens it will likely indicate an ABI break.
+*Minor* versions are released regularly, typically every 6 months. Minor
+versions contain new features, deprecations, and removals of previously
+deprecated code. *Bugfix* releases are made even more frequently; they do not
+contain any new features or deprecations.
+
+It is important to know that NumPy, like Python itself and most other
+well-known scientific Python projects, does **not** use semantic versioning.
```

Comment:
No dash in "well known", likewise in "forward compatible", "backward compatible", and  "build time", although the latter sometimes has a dash. I suspect all these usages are still in flux and may change, aka, runtime.

Reply:
thanks, fixed except for build-time (it made at least one sentence quite unclear with removed dash)